src/anemoi/transform/filters/matching.py

```diff
@@ -211,16 +223,32 @@ def forward(self, data: ekd.FieldList) -> ekd.FieldList:
             Transformed data.
         """
         args = []
+        match self.return_inputs:
+            case "all":
+                returned_input_list = self.forward_arguments
+            case "none":
+                returned_input_list = []
+            case _:
+                if not isinstance(self.return_inputs, list):
+                    raise ValueError("Return inputs must be 'all', 'none', or List[str].")
+                if not set(self.return_inputs) <= set(self.forward_arguments):
+                    raise ValueError("Returned input names must be in the forward arguments")
```

Comment:
I would add the `self.forward_arguments` to the error statement so it's more explicit/clear from the user point of view?